code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,356 @@
+	else
+		new /obj/effect/decal/cleanable/blood(get_turf(A))
+
+/obj/item/slimecross/destabilized/green
+	colour = "green"
+	effect_desc = "Transfers some of the thrower's wounds to the hit creature."
+
+/obj/item/slimecross/destabilized/green/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A) && isliving(thrower))
+		var/mob/living/LTarget = A
+		var/mob/living/LThrower = thrower
+		var/damageToDeal
+		if(iscarbon(LThrower))
+			var/mob/living/carbon/CThrower = LThrower
+			var/damageType = findBiggestLoss(CThrower)
+			damageToDeal = min(CThrower.get_damage_amount(damageType), 25)
+			CThrower.apply_damage_type(-25, damageType)
+			if(iscarbon(LTarget))
+				var/mob/living/carbon/CTarget = LTarget
+				CTarget.apply_damage_type(damageToDeal, damageType)
+				return
+			LTarget.adjustBruteLoss(damageToDeal)
+			return
+		damageToDeal = -(LThrower.adjustBruteLoss(-25))
+		LTarget.adjustBruteLoss(damageToDeal)
+
+/obj/item/slimecross/destabilized/green/proc/findBiggestLoss(mob/living/carbon/C)
+	var/damageType = BRUTE
+	var/maxDamage = C.get_damage_amount(BRUTE)
+	var/burnDamage = C.get_damage_amount(BURN)
+	if(burnDamage > maxDamage)
+		maxDamage = burnDamage
+		damageType = BURN
+	var/toxinDamage = C.get_damage_amount(TOX)
+	if(toxinDamage > maxDamage)
+		maxDamage = toxinDamage
+		damageType = TOX
+	var/oxyDamage = C.get_damage_amount(OXY)
+	if(oxyDamage > maxDamage)
+		maxDamage = oxyDamage
+		damageType = OXY
+	var/cloneDamage = C.get_damage_amount(CLONE)
+	if(cloneDamage > maxDamage)
+		maxDamage = cloneDamage
+		damageType = CLONE
+	var/staminaDamage = C.get_damage_amount(STAMINA)
+	if(staminaDamage > maxDamage)
+		maxDamage = staminaDamage
+		damageType = STAMINA
+	var/brainDamage = C.get_damage_amount(BRAIN)
+	if(brainDamage > maxDamage)
+		maxDamage = brainDamage
+		damageType = BRAIN
+	return damageType
+
+/obj/item/slimecross/destabilized/pink
+	colour = "pink"
+	reusable = TRUE //Hugs are hard enough to come by as is.
+	effect_desc = "Allows the thrower to hug another individual from far away. Just as the joyful feeling of hugs lasts forever, so do these extracts!"
+
+/obj/item/slimecross/destabilized/pink/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(!isliving(A) || !iscarbon(thrower))
+		return FALSE //Don't play the noise unless it lands a hit on a eligible mob.
+	var/mob/living/carbon/CThrower = thrower
+	if(iscarbon(A))
+		var/mob/living/carbon/CTarget = A
+		CTarget.help_shake_act(CThrower)
+	else
+		var/mob/living/LTarget = A
+		var/oldIntent = CThrower.a_intent
+		CThrower.a_intent = "help"
+		LTarget.attack_hand(CThrower)
+		CThrower.a_intent = oldIntent
+
+/obj/item/slimecross/destabilized/gold
+	colour = "gold"
+	effect_desc = "Causes goliath-like tendrils to shoot up out of the ground at the tile that was hit."
+
+/obj/item/slimecross/destabilized/gold/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	new /obj/effect/temp_visual/goliath_tentacle(get_turf(A), thrower, FALSE)
```

Comment:
Feel free to make this goliath_tentacle/original, a single tile tentacle is a bit too weak.